src/index.js

```diff
@@ -117,12 +119,10 @@ function allFederalHolidaysForYear(year = (new Date().getFullYear())) {
 }
 
 module.exports = {
-  isAHoliday(date = new Date()) {
+  isAHoliday(date = new Date(), federalReserveMode = false) {
     let isHoliday = false;
 
-    // Get this year and next, to handle the case where December 31 is the
-    // observed holiday for January 1 of the following year.
-    const allForYear = allFederalHolidaysForYear(date.getFullYear()).concat(allFederalHolidaysForYear(date.getFullYear() + 1));
```

Comment:
Can you explain why you took this out?  This was put in to handle the case where December 31 is a Friday, it should be a holiday, but the following January 1 would not be in the `allForYear` list because it's for the next year.  If you've found a better solution elsewhere in the code, I'm all for it.  :)

Reply:
This is a great example of how unit tests would be helpful. I guess the nearest example of this case is `federalReserveMode=false` on New Year's Day, Friday `2021-12-31`. Sorry, when I was in there I couldn't infer which case you were trying to cover.

To answer your question, I have a database of federal reserve holidays between `2013-01-01` and `2028-12-31`. I had to put this code back in and run the script against that database again in order to remember the test case.

It looks like I took this out because it breaks federalReserveMode for the following dates:

- 2015-12-26T00:00:00.000Z
- 2021-12-26T00:00:00.000Z
- 2022-12-25T00:00:00.000Z

In the case of `2015-12-26T00:00:00.000Z` this is because Christmas is on `2016-12-26`. If we add 2016's holidays to the list, the code will match 12-26 from 2016 because there is no year qualifier here: https://github.com/18F/us-federal-holidays/blob/c19833be58a515d5ae46c9bf2c04a2ead77ca784/src/index.js#L129
or here: https://github.com/18F/us-federal-holidays/blob/c19833be58a515d5ae46c9bf2c04a2ead77ca784/src/index.js#L146

Maybe the best solution is to concat just the first entry from the next year?